### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by dev-fredericfox** [community/community#40022] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I worked with copilot for the past couple of months but tonight it randomly stopped working with this weird error message:
`GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."` 

I tried:
- logging out, and logging back in
- uninstalling the extension and reinstalling it
- uninstalling VSCode
- using a VPN
- loggin in to a different github account and back to my account with the sub
- Nightly and normal Build of the extension

It's a shame I have been customer from day 1 but will cancel my sub if I can't fix it.

Copilot Output:
```
[INFO] [auth] [2022-11-24T22:05:31.409Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:05:31.410Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-24T22:05:36.069Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:05:36.070Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-24T22:05:37.400Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:05:37.401Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-24T22:05:38.681Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:05:38.681Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[INFO] [auth] [2022-11-24T22:07:34.700Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:07:34.703Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
[ERROR] [default] [2022-11-24T22:09:43.021Z] GitHub Copilot could not connect to server. Extension activation failed: "User did not consent to login."
[INFO] [auth] [2022-11-24T22:10:03.488Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-24T22:10:03.489Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by nathan-roberts** [comment]
I'm having the same issue

**Reply by Wxh16144** [reply]
I also met it.

**Comment by ravhirizaldi** [comment]
same here to me! It's so frustating. I getting billed today and it's not working

**Comment by si9ma** [comment]
same here to me!

**Comment by blackshot** [comment]
same here! i paid a yearly plan...

**Comment by cainthebest** [comment]
im having the same issue above, tried everything and all i get is this 
```
[INFO] [auth] [2022-11-25T02:51:47.744Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-25T02:51:47.747Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
```

**Comment by lalalune** [comment]
+1 for me

plz i am lost without my ai superpowers

**Comment by eniko1556** [comment]
+1 for me

**Comment by DaemonStark** [comment]
I'm having the same issue, any outage here? how do we know if the github support is looking at these forums or not? Can anyone from github org confirm on this? @github-staff - is this the right mention?

**Comment by KrystilizeNevaDies** [comment]
Same issue here. It is very frustrating to not get access despite paying for the product.

**Comment by pablodz** [comment]
Same here!

**Comment by adityarizqi** [comment]
yeah me to

**Comment by joshuawalker** [comment]
We're looking into it now! 🔍 

https://www.githubstatus.com/incidents/69flmdbxhtsk

**Reply by KrystilizeNevaDies** [reply]
🙏

**Reply by btfranklin** [reply]
With all due respect, this should not be marked as "Resolved". It is definitely not resolved.

**Comment by davecheney** [comment, marked as answer]
Hello. I'm sorry you had difficulty accessing Copilot, please accept my apologies on behalf of the Copilot team. This was caused by a bad deploy which affected a small percentage of Copilot users. The change has been rolled back and you should be able to use Copilot again.

**Reply by ahmedyounes** [reply]
Appreciated! but hurry please I'm addicted to it

**Reply by esQmo** [reply]
same problem here!

**Reply by AndyManettas** [reply]
Hi Dave, the issue has returned.

**Comment by ahmedyounes** [comment]
![Screen Shot 2022-11-26 at 2 15 06 AM](https://user-images.githubusercontent.com/28989583/204065132-cdb86c57-39d5-4bb3-9832-2f8de272a8c4.png)
im on yearly plan

**Comment by dialformonkey** [comment]
Affected here as well.

**Comment by ChenghaoMou** [comment]
Same here.

**Comment by btfranklin** [comment]
I am still seeing this exact issue. From my VSCode log:

[INFO] [auth] [2022-11-26T00:17:03.653Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2022-11-26T00:17:03.655Z] GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

Please observe the timestamps. I have uninstalled/reinstalled Copilot extensions, including the "Nightly" variant, to trigger a re-auth into my account, which did not help.

**Comment by adambiggs** [comment]
I'm still having the same issue too

**Comment by manychois** [comment]
For me the situation was it failed yesterday morning, then worked at night, then failed again this morning...

**Reply by mil0sh** [reply]
Same for me

**Comment by bobaikato** [comment]
Just got this issue. I hate the "withdrawal syndrome" I am going right through. Someone help!

**Comment by Saissaken** [comment]
Still happening here! Need my AI superpowers back

**Comment by xuedaobian** [comment]
same issue

**Reply by xuedaobian** [reply]
<img width="461" alt="image" src="https://user-images.githubusercontent.com/38198170/204066808-6e8a3850-dda4-4b9f-9791-716b539860b8.png">

**Comment by cfox4ever** [comment]
I'm still having the same issue too

**Comment by DeveloperMartin** [comment]
same issue 
![image](https://user-images.githubusercontent.com/43618471/204067285-72ac2142-60ce-4fa1-bb52-8754d061bacb.png)

**Reply by DeveloperMartin** [reply]
A possible solution can be by touching the "retry" button to log in using the generated unique code and that way the login worked for me.

**Comment by btfranklin** [comment]
It looks like they're actively working the issue right now: https://www.githubstatus.com/incidents/3208bb8blz5k

**Reply by btfranklin** [reply]
Just now, mine appears to be working again! Thanks CoPilot team!

**Comment by chezhe** [comment]
+1

**Comment by JohniMIEP** [comment]
It still doesnt work even when github says that its resolved
